llvm/test/Transforms/LoopVectorize/partial-reduce-dot-product.ll

```diff
@@ -0,0 +1,272 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt -passes=loop-vectorize -force-vector-interleave=1 -S < %s | FileCheck %s
+
+target datalayout = "e-m:e-i8:8:32-i16:16:32-i64:64-i128:128-n32:64-S128"
+target triple = "aarch64-none-unknown-elf"
```

Comment:
I think this should either be moved to the AArch64 subdirectory or not target AArch64. You might also be missing necessary attributes for AArch64 (+sve? +dotprod?) needed to transform this.